### PR TITLE
Add sequence example support to scio-tensorflow

### PR DESCRIPTION
**Pull request by martinbomio**
Adds support for 9SequenceExample](https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/core/example/example.proto) to scio tensorflow.
Addresses https://github.com/spotify/scio/issues/1659.

### CODE REVIEW

File: scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TFRecordIO.scala [hunk 1]
```diff
@@ -88,6 +87,26 @@ object TFExampleIO {
   val WriteParam = TFRecordIO.WriteParam
 }
 
+final case class TFSequenceExampleIO(path: String) extends ScioIO[SequenceExample] {
```
Comment (martinbomio) [inline scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TFRecordIO.scala]: created a new `ScioIO` for the sequence example following how it was done with the exampleIO and the tfrecordIO, but both the `TFSequenceExampleIO ` and `TFExampleIO` can be deleted and the `fRecordSequenceExampleFile` and `fRecordExampleFile` methods can use the `TFRecordIO ` and just map and create the *Examples from bytes. Same applies to the write functions.

File: scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala [hunk 2]
```diff
@@ -449,3 +449,23 @@ final class TFRecordSCollectionFunctions[T <: Array[Byte]](private val self: SCo
...
+  private val self: SCollection[T])
+    extends AnyVal {
+
+  /**
+   * Saves this SCollection of `org.tensorflow.example.SequenceExample` as a TensorFlow
+   * TFRecord file.
+   *
+   * @return
+   */
+  def saveAsTfSequenceExampleFile(
```
Comment (regadas) [inline scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala]: this is in line with the rest of the stuff but I'm getting the idea that we probably could rename this methods to just `saveAsTfRecordFile`. This would be on another PR but @martinbomio  wdyt?

Comment (brianmartin) [reply scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala]: +1

Comment (martinbomio) [reply scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala]: sounds good to me!

Comment (martinbomio) [reply scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala]: created https://github.com/spotify/scio/pull/1762
